Calculator.py

```diff
@@ -17,3 +17,10 @@ def average(self, nums):
     nums_sum = self.sum(nums)
     result = nums_sum / l
     return result
+  def cm_to_metres(self, nums):
+    results_list = []
+    for i in nums:
+      metres = i/100
+      results_list.append(str(i) + "cm = " + str(metres) + "m")
```

Comment:
Calculator class is responsible for only calculating. Not formatting it for user. That's the role of `cli.py`. Here just return the result as an array. Also use Python3 map function to transform list of centimetres to list of metres.

Reply:
> use Python3 map function to transform list of centimetres to list of metres.
I have 2 doubts here:
1. How would I use `map(function, iterable)` here? Since `func` would be another function which taken in each cm value and outputs the m value.
2. Instead of using `map()` can I do the following? 
```
def cm_to_m(self, nums):
    return([i/100 for i in nums])
```